src/poly/commitment/prover.rs

```diff
@@ -220,15 +220,11 @@ fn parallel_generator_collapse<C: CurveAffine>(
     challenge_inv: C::Scalar,
 ) {
     let len = g.len() / 2;
-    let (mut g_lo, g_hi) = g.split_at_mut(len);
-
-    parallelize(&mut g_lo, |g_lo, start| {
```

Comment:
We still want to do the parallelization exactly as we've done before, it's just that we want to replace the line

```rust
tmp.push(((*g_lo) * challenge_inv) + &((*g_hi) * challenge));
```

with

```rust
tmp.push(small_multiexp(&[challenge_inv, challenge], &[*g_lo, *g_hi]));
```